### PR TITLE
Address some issues in acpi_tables.py

### PR DESCRIPTION
**Pull request by frinzell**
- Fix byte parsing
- Rename 'type' named variables
- Fix a counter
- Initialize a variable that may not get set
- Shorten some lines
- Remove some commented code

Signed-off-by: Frinzell, Aaron <aaron.frinzell@intel.com>

### CODE REVIEW

File: chipsec/hal/acpi_tables.py [hunk 1]
```diff
@@ -1333,7 +1337,7 @@ def parseAMCS(self, table_content, type):
         currBank = 0
         while currBank < numHardwareBanks:
             self.machineBankParser(table_content[48 + currBank * 28:48 + (currBank + 1) * 28])
-            numHardwareBanks == 1
+            numHardwareBanks += 1
```
Comment (npmitche) [inline chipsec/hal/acpi_tables.py:1340]: Should it not be `currBank += 1`?

File: chipsec/hal/acpi_tables.py [hunk 2]
```diff
@@ -1928,14 +1934,22 @@ def parseMAP(self, tableLen, table_content):
...
-        ramDiskVirtualDiskVolGUID = [int('0x77ab535a', 16), int('0x45fc', 16), int('0x624b', 16), int('0x55', 16), int('0x60', 16), int('0xf7', 16), int('0xb2', 16), int('0x81', 16), int('0xd1', 16), int('0xf9', 16), int('0x6e', 16)]
-        ramDiskVirtualCDVolGUID = [int('0x3d5abd30', 16), int('0x4175', 16), int('0x87ce', 16), int('0x6d', 16), int('0x64', 16), int('0xd2', 16), int('0xad', 16), int('0xe5', 16), int('0x23', 16), int('0xc4', 16), int('0xbb', 16)]
-        ramDiskVirtualDiskPersisGUID = [int('0x5cea02c9', 16), int('0x4d07', 16), int('0x69d3', 16), int('0x26', 16), int('0x9f', 16), int('0x44', 16), int('0x96', 16), int('0xfb', 16), int('0xe0', 16), int('0x96', 16), int('0xf9', 16)]
-        ramDiskVirtualCDPersisGUID = [int('0x08018188', 16), int('0x42cd', 16), int('0xbb48', 16), int('0x10', 16), int('0x0f', 16), int('0x53', 16), int('0x87', 16), int('0xd5', 16), int('0x3d', 16), int('0xed', 16), int('0x3d', 16)]
+        volitileMemGUID = [int('0x7305944f', 16), int('0xfdda', 16), int('0x44e3', 16), int('0xb1', 16), int('0x6c', 16),
```
Comment (npmitche) [inline chipsec/hal/acpi_tables.py:1937]: Why are we parsing ints here? Can't we just change `int('0x7305944f', 16)` to `0x7305944f`?

**Review by npmitche — CHANGES_REQUESTED** [review]

File: chipsec/hal/acpi_tables.py (hunk 1 above)
Comment (frinzell) [reply chipsec/hal/acpi_tables.py:1340]: yup

File: chipsec/hal/acpi_tables.py (hunk 2 above)
Comment (frinzell) [reply chipsec/hal/acpi_tables.py:1937]: You were the last person to touch this code section. :p  It could.

Comment (npmitche) [reply chipsec/hal/acpi_tables.py:1937]: In my defense, I didn't look at the file, just formatted it. 😇

**Review by npmitche — APPROVED** [review]
Thanks Aaron!

**Review by Sae86 — APPROVED** [review]

**Pull request merged**